### ISSUE TITLE
Can't get local plugin to install

### ISSUE THREAD

**Issue by kevin8181** [saltcorn/saltcorn#1115]
I can't get a plugin to install locally- see pic below

![image](https://user-images.githubusercontent.com/66894759/156071068-7cb54057-4512-405d-a37f-e69cb18a2d2a.png)

What's going wrong here?

**Comment by kevin8181** [comment]
it does the same thing for other plugins

installed with the ubuntu quick install script

**Comment by glutamate** [comment]
@kevin8181 the path has to be absolute, so try `... -d ~/fullcalendar`

I have updated the CLI help to describe this

**Comment by kevin8181** [comment]
what's the easiest way to test new plugin code? is there something better than re-installing the plugin and restarting the instance every time? I'm not able to get anything installed through the admin ui with local, git, or github, but got it to install over cli

@glutamate

**Comment by glutamate** [comment]
Yes, there is a cli command `localize-plugin` which makes this much easier. So I would:

1) install the plug-in as normal in the plugins tab
2) git clone the repo
3) run e.g. for the "badges" plugin: `saltcorn localize-plugin badges ~/badges`
